src/reaper_osara.cpp

```diff
@@ -922,6 +926,7 @@ PostCommand POST_COMMANDS[] = {
 	{40696, postRenameTrack}, // Track: Rename last touched track
 	{40175, postToggleItemMute}, // Item properties: Toggle mute
 	{40626, postSetSelectionEnd}, // Time selection: Set end point
+	{ 40917, postToggleMasterMono}, //Master track: Toggle stereo/mono (L+R)
```

Comment:
nit (for consistency): Remove space after left brace. Add space between "//" and comment.